packages/liveblocks-redux/src/index.ts

```diff
@@ -353,14 +376,10 @@ function enterRoom(roomId: string): {
   };
 }
 
-function leaveRoom(roomId: string): {
+function leaveRoom(): {
```

Comment:
Is it a breaking change?

Reply:
Nope! Only at the type level it is. TypeScript will not like it if you call `leaveRoom("my-room")` but at runtime it works. I considered doing this instead:

```tsx
function leaveRoom(
  /** @deprecated No longer needed */
  _roomId?: string
) {
}
```

But outside of TypeScript, this isn't a breaking change.